pkg/blob/controllerserver_test.go

```diff
@@ -2133,3 +2133,42 @@ func TestGetAzcopyAuth(t *testing.T) {
 		t.Run(tc.name, tc.testFunc)
 	}
 }
+
+func TestControllerModifyVolume(t *testing.T) {
+	d := NewFakeDriver()
+	ctx := context.Background()
+	
```

Comment:
Error: pkg/blob/controllerserver_test.go:2140:1: File is not properly formatted (gofmt)